src/doc/rustc/src/platform-support/netbsd.md

```diff
@@ -34,7 +34,7 @@ are built for NetBSD 8.x but also work on newer OS versions).
 ## Designated Developers
 
 - [@he32](https://github.com/he32), `he@NetBSD.org`
-- [NetBSD/pkgsrc-wip's rust](https://github.com/NetBSD/pkgsrc-wip/blob/master/rust/Makefile) maintainer (see MAINTAINER variable). This package is part of "pkgsrc work-in-progress" and is used for deployment and testing of new versions of rust
+- [NetBSD/pkgsrc-wip's rust](https://github.com/NetBSD/pkgsrc-wip/blob/master/Makefile) maintainer (see MAINTAINER variable). This package is part of "pkgsrc work-in-progress" and is used for deployment and testing of new versions of rust
```

Comment:
https://github.com/NetBSD/pkgsrc-wip/blob/master/rust185/Makefile is probably a better link. The top-level Makefile is completely unrelated to rust.